BDArmory.Core/Module/HitpointTracker.cs

```diff
@@ -53,6 +53,8 @@ public class HitpointTracker : PartModule, IPartMassModifier, IPartCostModifier
         public float HullCostAdjust = 0f;
         double resourceCost = 0;
 
+        double resourceCost = 0;
```

Comment:
This is a duplicate too.

Reply:
Oh, I know what's going on - when I originally pushed this, it was from partch-2, not dev, so loading the dev PR needed a fetch -> push, which resulted in a merge
fixed.